packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -27,6 +27,7 @@ import nock from 'nock';
 import * as sinon from 'sinon';
 
 import { advanceTime } from '../../../tests/helpers';
+import { createMockInternalAccount } from '../../accounts-controller/src/tests/mocks';
```

Comment:
Similarly here, should it be calling the method from the directory? Maybe we should move these shared methods to the `tests` directory